cmd/collapsed-kube-commit-mapper/main.go

```diff
@@ -74,6 +74,10 @@ func main() {
 	upstreamBranch := flag.String("upstream-branch", "", "the k8s.io/kubernetes branch (fully qualified e.g. refs/remotes/origin/master) used as the filter-branch basis")
 	showMessage := flag.Bool("l", false, "list the commit message after the two hashes")
 
+	// repository flags used when the repository is not k8s.io/kubernetes
+	repoName := flag.String("repo-name", "kubernetes", "the name of the source repository")
```

Comment:
would make these mandatory

Reply:
validate if not empty you mean? are you ok with defaulting? in that case we don't have to change anything

Reply:
cannot forsee how many changes this really needs. But we have to pass through the non-kubernetes repo anyway. So let's use the same code path(s) for the kube case. I.e. no defaulting, but an error message if empty.